kubernetes/include/generic.h

```diff
@@ -0,0 +1,31 @@
+#ifndef __GENERIC_H__
+#define __GENERIC_H__
+
```

Comment:
Could you please add the macro for c++ support ?

```c
#ifdef  __cplusplus
extern "C" {
#endif

#ifdef  __cplusplus
}
#endif
```

Reply:
done.